nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/test/java/org/apache/nifi/integration/FrameworkIntegrationTest.java

```diff
@@ -368,7 +368,6 @@ public Collection<FlowFileQueue> getAllQueues() {
         flowController.initializeFlow(queueProvider);
     }
 
-    @After
```

Comment:
It looks like `AfterEach` should be added to this method, or it should be consolidated with `shutdown()`.